src/sagemaker/jumpstart/model.py

```diff
@@ -321,7 +323,10 @@ def _create_sagemaker_model(self, *args, **kwargs):  # pylint: disable=unused-ar
             kwargs: Keyword arguments coming from the caller. This class does not require
                 any so they are ignored.
         """
-        if self.model_package_arn:
+
+        # if the user inputs a model artifact uri, do not use model package arn to create
+        # inference endpoint.
+        if self.model_package_arn and not self._model_data_is_set:
```

Comment:
btw, this is required for JS in Studio, since we use the `JumpStartModel` class to deploy fine-tuned models

Reply:
Just checking: Is there a UT in tests/unit/sagemaker/jumpstart/model/test_model.py that needs updating ?